policies/trigger-retries.rego

```diff
@@ -0,0 +1,15 @@
+package spacelift
+
+# You can read more about trigger policies here:
+#
+# https://docs.spacelift.io/concepts/policy/trigger-policy
+
+trigger[stack.id] {
+  stack := input.stack
+  input.run.state == "FAILED"
+  input.run.type == "TRACKED"
+  is_null(input.run.triggered_by)
```

Comment:
Is `input.run.triggered_by` ever `null`? What would cause that? Could we use Tags somehow to prevent a loop?

Reply:
not sure, I copied that policy from Spacelift docs

Reply:
Answer from Spacelift support:
> `triggered_by` is only set if the run was triggered by a user or by a trigger policy.
So it will be null, for example, if the Run is caused by a commit push and the corresponding push policy.